src/Benchmarks.ClientJob/Statistics.cs

```diff
@@ -19,6 +19,7 @@ public class Statistics
         public double Latency90Percentile { get; set; }
         public double Latency99Percentile { get; set; }
         public double TotalRequests { get; set; }
+        public double DeltaRequests { get; set; }
```

Comment:
Remind me what this is

Reply:
This is not needed anymore. I was going to add this field to differentiate between total requests for the current job and total requests across jobs but not anymore